### PR TITLE
Actually Balanced Vendor Prices

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
generally kicks vendor prices in the teeth
closes #12435

## Why It's Good For The Game
Fuck vendorconomy but I'm also too lazy to figure out how to remove it so I made the prices pretty much negligible for most things

## Changelog
:cl:
balance: rebalanced all the vendor prices
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Neat!

File: code/modules/vending/liberation.dm
Comment (Poojawa) [inline code/modules/vending/liberation.dm:31]: bUt ThATs SoCiALiSm

**Review by deathride58 — DISMISSED** [review]
Yakumo making good changes for once? Nice.

File: code/modules/vending/liberation.dm
Comment (silicons) [reply code/modules/vending/liberation.dm:31]: based

**Comment by YakumoChen** [comment]
alright guess I gotta go through every single fucking vendor and set a bunch of prices because nothing can be default

**Comment by YakumoChen** [comment]
That was painful

File: code/modules/vending/clothesmate.dm
Comment (Ghommie) [inline code/modules/vending/clothesmate.dm]: slightly more. 40/100 should be fine.

File: code/modules/vending/coffee.dm
Comment (Ghommie) [inline code/modules/vending/coffee.dm:18]: 50 or so.

File: code/modules/vending/games.dm
Comment (Ghommie) [inline code/modules/vending/games.dm:16]: the extra price is fine as if.

File: code/modules/vending/kinkmate.dm
Comment (Ghommie) [inline code/modules/vending/kinkmate.dm:44]: Same prices as the clothesmate, no matter what.

File: code/modules/vending/liberation.dm
Comment (Ghommie) [inline code/modules/vending/liberation.dm:32]: The few maploaded liberation vendors should already be free as they aren't initialized on the station z levels... but if you are in dire needs of a freeloader version of this rare vendor, please make a subtype, or VV the thing in game.

File: code/modules/vending/liberation_toy.dm
Comment (Ghommie) [inline code/modules/vending/liberation_toy.dm:29]: Same as above.

File: code/modules/vending/magivend.dm
Comment (Ghommie) [inline code/modules/vending/magivend.dm:20]: Idem. These should never be found on the station z level.

File: code/modules/vending/toys.dm
Comment (Ghommie) [inline code/modules/vending/toys.dm:29]: Same as above, yet these ones can be obtained in a normal round with some efforts.

File: code/modules/vending/snack.dm
Comment (Ghommie) [inline code/modules/vending/snack.dm]: 20/80

File: code/modules/vending/youtool.dm
Comment (Ghommie) [inline code/modules/vending/youtool.dm:25]: 20. Still lower than the average greyman paycheck.

File: code/modules/clothing/gloves/color.dm
Comment (Ghommie) [inline code/modules/clothing/gloves/color.dm]: I'd say `PAYCHECK_MEDIUM * STARTING_PAYCHECKS + 25` (900) is a better price, so the average non-greyshirt will always be a few credits shy from buying those gloves roundstart. :smiling_imp:

File: code/modules/food_and_drinks/food/snacks_vend.dm
Comment (Ghommie) [inline code/modules/food_and_drinks/food/snacks_vend.dm]: 30?

File: code/modules/food_and_drinks/food/snacks_vend.dm
Comment (Ghommie) [inline code/modules/food_and_drinks/food/snacks_vend.dm]: 10-15

File: code/modules/vending/autodrobe.dm
Comment (Ghommie) [inline code/modules/vending/autodrobe.dm]: 60, slightly more expensive than the clothesmate.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Many parts this PR are way too generous in my opinion, but it does take care of a few gripes I had while porting goofconomy.

Yet I couldn't help but notice this PR was made at the distance of weeks, shortly after the feature was merged on a downstream, so please bear with me a moment while I code in a few configs before you make vendor prices bite the dust.

File: code/modules/vending/games.dm
Comment (YakumoChen) [reply code/modules/vending/games.dm:16]: There is nothing default priced in the vendor premo regardless.

Comment (Ghommie) [reply code/modules/vending/games.dm:16]: Ok then.

File: code/modules/vending/kinkmate.dm
Comment (YakumoChen) [reply code/modules/vending/kinkmate.dm:44]: I will not be budging on kinkmate. Kinkmate appeals to assistants who have very tight paychecks and putting ERP items out of their budget, preventing them from fucking off to dorms because of an arbitrary economy feature, isnt acceptable.

I have half a mind to make the items free, and if I'm forced to price the items up I will make the maint kinkmate on maps a freebie.

**Comment by YakumoChen** [comment]
My prices are balanced around the assistant starting budget (25*5=125), given they're the only job with a major budget constraint currently.

Vendor economy isnt a feature maintainers can agree on (afaik) so I would rather aim low than aim for the middle which just locks assistants out from basic RP / snowflake items

File: code/modules/vending/youtool.dm
Comment (YakumoChen) [reply code/modules/vending/youtool.dm:25]: Tools are literally, _literally_ everywhere, more abundant than food, and can be printed from autolathes, i see no reason why a higher / same price than food is needed when food is a greater scarcity.

File: code/modules/vending/kinkmate.dm
Comment (Ghommie) [reply code/modules/vending/kinkmate.dm:44]: I have no plan to make the server any more appealing to people who merely wish to wind up in a rendezvous scene, and this vendor is not strictly necessary for that either.

On top of that, those players usually do little beside sitting at the bar, typing stuff and getting an article of clothing they couldn't have from the loadout system, they'll likely save up a hundred or two for that anyway.
TL;DR: I ain't budging either.

**Comment by Ghommie** [comment]
I have no plan to make the game revolve around "off-duty employeer" players idly typing stuff to each other and slacking around.

File: code/modules/vending/kinkmate.dm
Comment (YakumoChen) [reply code/modules/vending/kinkmate.dm:44]: I am not interested in a system with the purpose of being vindictive to ERPers by withholding them flavor / fetish items due to some kind of code grudge. There will be no change.

File: code/modules/vending/coffee.dm
Comment (YakumoChen) [reply code/modules/vending/coffee.dm:18]: The "premium" items consist of chocolate and milk, so I don't really think it's needed.

File: code/modules/vending/toys.dm
Comment (YakumoChen) [reply code/modules/vending/toys.dm:29]: Non-issue. The toylathe prints all of these, for free, with less effort to obtain than a stocked toy vendor.

File: code/modules/vending/liberation.dm
Comment (YakumoChen) [reply code/modules/vending/liberation.dm:32]: AMERICAN JUSTICE HAS NO PRICE OTHER THAN THE BLOOD AND SWEAT OF OUR COMPATRIOTS

**Comment by Poojawa** [comment]
When did Ghommie get made head coder?

**Comment by YakumoChen** [comment]
The prices have been raised slightly, but I do not intend to go higher.

File: code/modules/vending/youtool.dm
Comment (Ghommie) [reply code/modules/vending/youtool.dm:25]: You are tempting me to remove some toolboxes from maps to justify the prices.

File: code/modules/vending/toys.dm
Comment (Ghommie) [reply code/modules/vending/toys.dm:29]: I'm going to nerf the toylathe to hell, it has been a nuisance anyway.

File: code/modules/vending/kinkmate.dm
Comment (Ghommie) [reply code/modules/vending/kinkmate.dm:44]: It seems we are reached a deadlock then. I'm off to make my own PR, no gain in being stubborn here.

**Pull request closed without merging**
